### PR TITLE
Release 1.0 node contribution - More OS support added

### PR DESCRIPTION
**Pull request by eqfarhad**
List of tested and added operating systems:

- CentOS 7
- Red Hat Enterprise Linux (RHEL) 7
- Fedora 25+
- HypriotOS v1.0.1+

- [ ] The ID of real HypriotOS should get check

### CODE REVIEW

File: pkg/controller/v1alpha/nodecontribution/handler.go [hunk 1]
```diff
@@ -774,6 +774,9 @@ func getInstallCommands(conn *ssh.Client, hostname string, kubernetesVersion str
 		}
 		if ubuntu, _ := regexp.MatchString("ID=\"ubuntu\".*|ID=ubuntu.*", string(output[:])); ubuntu {
 			dockerInstallation = ubuntuCommands
+		} else if hypriotOS, _ := regexp.MatchString("ID=\"hypriotOS\".*|ID=hypriotOS.*", string(output[:])); hypriotOS {
+			// The ID of hypriotOS should become checked again
+			dockerInstallation = []string{""}
```
Comment (bsenel) [inline pkg/controller/v1alpha/nodecontribution/handler.go]: Do we have an alternative for `dockerInstallation = []string{""}` like `dockerInstallation = nil` here?

**Review by bsenel — CHANGES_REQUESTED** [review]
Please just check the issue I pointed out then we can merge this.

File: pkg/controller/v1alpha/nodecontribution/handler.go (hunk 1 above)
Comment (eqfarhad) [reply pkg/controller/v1alpha/nodecontribution/handler.go]: okay, i will check it now

Comment (eqfarhad) [reply pkg/controller/v1alpha/nodecontribution/handler.go]: I checked and it's working well with nil. so i changed it.

**Review by bsenel — APPROVED** [review]
It is OK now, congrats! 🎉

**Pull request merged**